src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4406,24 +4427,28 @@ private BoundExpression BindClassCreationExpression(ObjectCreationExpressionSynt
         /// </summary>
         private BoundExpression MakeConstructorInvocation(
             NamedTypeSymbol type,
-            ImmutableArray<BoundExpression> arguments,
+            ArrayBuilder<BoundExpression> arguments,
+            ArrayBuilder<RefKind> refKinds,
             SyntaxNode node,
-            BindingDiagnosticBag diagnostics)
+            BindingDiagnosticBag diagnostics,
+            out bool overloadResolutionSucceeded)
```

Comment:
As below, is this required or is the `LookupResultKind` on the returned expression enough to decide?